elm.json

```
@@ -3,16 +3,15 @@
     "name": "Holmusk/elmoji",
     "summary": "A tabbed general-purpose emoji picker",
     "license": "BSD-3-Clause",
-    "version": "1.0.2",
+    "version": "1.0.4",
```

just reverting this bump

```suggestion
    "version": "1.0.2",
```